app/questionnaire/questionnaire_schema.py

```diff
@@ -94,11 +96,75 @@ def __init__(
         self._populate_answer_dependencies()
         self._populate_when_rules_section_dependencies()
         self._populate_calculated_summary_section_dependencies()
+        self._populate_min_max_for_numeric_answers()
 
     @cached_property
     def answer_dependencies(self) -> ImmutableDict[str, set[AnswerDependent]]:
         return ImmutableDict(self._answer_dependencies_map)
 
+    @cached_property
+    def min_and_max_map(self) -> ImmutableDict[str, str | dict[str, str]]:
+        return ImmutableDict(self._min_and_max_map)
+
+    def _set_min_max_value_for_answer(
```

Comment:
Function is misnamed, it doesn't set anything?. `get_mix_max...`

Reply:
It was setting the longest string value but I changed it to return it instead. Renamed